src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static ReaderService readerService;
+    private static final String FILE_PATH_FIRST = "src/test/resources/notValidFile.csv";
+    private static final String FILE_PATH_SECOND = "src/test/resources/validFile.csv";
+    private static final String FILE_PATH_THIRD = "src/test/resources/emptyFile.csv";

```

Comment:
```suggestion
    private static final String NOT_VALID_FILE_PATH = "src/test/resources/notValidFile.csv";
    private static final String VALID_FILE_PATH = "src/test/resources/validFile.csv";
    private static final String EMPTY_FILE_PATH= "src/test/resources/emptyFile.csv";

```
These names will be more informative.